runtime/src/Zynga.Protobuf.Runtime.Tests/EventTest.cs

```diff
@@ -1575,10 +1571,7 @@ public enum BodyOneofCase {
               break;
               case 2: {
                 if (body_ == null) body_ = new global::Com.Zynga.Runtime.Protobuf.EventTest.Types.NestedMessage();
-                if (e.Path.Count - 1 != pathIndex) 
-                  (body_  as zpr::EventRegistry)?.ApplyEvent(e, pathIndex + 1);
-                else
-                  body_   = global::Com.Zynga.Runtime.Protobuf.EventTest.Types.NestedMessage.Parser.ParseFrom(e.Data.ByteData);
+                body_  = global::Com.Zynga.Runtime.Protobuf.EventTest.Types.NestedMessage.Parser.ParseFrom(e.Data.ByteData);
```

Comment:
This doesn't look right, it seems like the NestedMessage is still event sourced, but the wrapping messaged doesn't  know it?